web/libs/datamanager/src/components/DataManager/Toolbar/LabelButton.jsx

```diff
@@ -84,30 +86,30 @@ export const LabelButton = injector(({ store, canLabel, size, target, selectedCo
   return canLabel ? (
     <Interface name="labelButton">
       <div>
-        <div style={{ display: "flex" }}>
+        <ButtonGroup>
           <Button
-            size={size}
+            size="small"
+            variant="primary"
+            look="outlined"
             disabled={disabled}
-            mod={{ size: size ?? "medium", look: "primary", disabled }}
             style={primaryStyle}
             onClick={onLabelAll}
           >
             Label {selectedCount ? selectedCount : "All"} Task{!selectedCount || selectedCount > 1 ? "s" : ""}
           </Button>
-          <Button
-            ref={triggerRef}
-            size={size}
-            mod={{ size: size ?? "medium", look: "primary", disabled }}
-            style={triggerStyle}
-            onClick={toggleOpen}
-            aria-label={"Toggle open"}
+          <Dropdown.Trigger
+            align="right"
+            content={
+              <Menu size="compact">
+                <Menu.Item onClick={onLabelVisible}>Label Tasks As Displayed</Menu.Item>
+              </Menu>
+            }
           >
-            {isOpen ? <IconChevron /> : <IconChevronDown />}
-          </Button>
-        </div>
-        <Button size={size} style={secondStyle} mod={{ size: size ?? "medium", disabled }} onClick={onLabelVisible}>
-          Label Tasks As Displayed
-        </Button>
+            <Button size="small" look="outlined" variant="primary" aria-label={"Toggle open"}>
+              <IconChevronDown />
```

Comment:
we're losing the visual toggle when the dropdown is opened - not a show stopper but would be nice to preserve it if possible

Reply:
i'm not sure if any of our dropdowns besides this one had it at all. i would prefer a more centralized solution. for now i believe it's good. and you still see the DD, pretty good indication that it's opened :)

Reply:
Fair enough, i do agree it should be done in a better way than possible at the moment. Perhaps to be addressed with the soon to come drop-down/popover overhaul